include/LightGBM/config.h

```diff
@@ -289,6 +289,11 @@ struct Config {
   // desc = random seed for ``feature_fraction``
   int feature_fraction_seed = 2;
 
+  // desc = use extremely randomized trees
+  // desc = if set to ``true``, when evaluating node splits LightGBM will check only one randomly-chosen threshold for each feature
+  // desc = can be used to deal with over-fitting
```

Comment:
Please add it here as well: https://github.com/microsoft/LightGBM/blob/master/docs/Parameters-Tuning.rst#deal-with-over-fitting.